### ISSUE TITLE
Create new "Fourm Posts" from TWAB announcment posts.

### ISSUE THREAD

**Issue by KGB33** [KGB33/RoboShpee#20]
TLDR:
  - Listen for TWAB announcements in `#d2-general`
  - Create post in `#d2-news` with title & link.
  - Respond to announcement message with link to news post.

---

Currently, Bungie blog posts/tweets are being broadcasted to `#d2-general`.
When a TWAB (TWID) is announced, RoboShpee should create a new forum post in `#d2-news` for the TWAB & link to it in a response to the announcement post.

Currently, I'm doing so manually, which looks like:

![image](https://github.com/KGB33/RoboShpee/assets/17833726/704248e2-65b0-4de7-bbbb-66b850a1dab0)
![image](https://github.com/KGB33/RoboShpee/assets/17833726/fc6bdd90-7c15-4112-ac34-764e2cb6054c)

The title for each TWAB post should be `f"TWAB - {subtitle}"`. For example, for the following TWID, the post title should be "TWAB - Ready for Iron Banner?". Discord dates each form post, so it should be left out of the title.

![image](https://github.com/KGB33/RoboShpee/assets/17833726/0487e551-d8cc-4a38-94ee-34ece469320f)
